### DISCUSSION TITLE
Sharing cache key refresh results among JuiceFS Hadoop SDK running in one NodeManager instead of refreshing individually

### DISCUSSION THREAD

**Discussion by cmmp6** [juicedata/juicefs#2867] [Ideas]
**What would you like to be added**:
Hadoop SDK refreshes cache key when initing cache and every 5 minute individually. I think it's better to share refresh results among JuiceFS Hadoop SDK running in one NodeManager than refreshing individually.

**Why is this needed**:
If there are many containers running with JuiceFS Hadoop SDK in NodeManager, refreshing cache key individually will generate many meta IO requests to cache disk. And it will be worse for shared cache dir, such as NFS or SMB.

**Comment by SandyXSD** [comment]
This is non-trivial, we have no plan for it.
